socs/agents/wiregrid_actuator/agent.py

```diff
@@ -242,6 +242,17 @@ def _insert_eject(
             self.log.error(msg)
             return False, msg
 
+        # Additional small moving to ensure the limit switch ON
+        # No limit switch check during this moving
+        # Only this function reflects pos/neg of <distance>.
+        # Please use +/- distance for insertion/ejection, respectively.
```

Comment:
Is this flipping a sign? `move_func` is `self._forward` if `is_insert`, and `self._forward` takes a positive distance always:
https://github.com/simonsobs/socs/blob/f40b3355d807b1094ef64dadc402cbd864282e5d/socs/agents/wiregrid_actuator/agent.py#L101-L105

So it seems like this is also moving in the typical expected direction to me.

Reply:
I mean that `move_func` (=`_forward` or `_backward`) will change the sign of `distance` automatically, but `self.actuator.move`, which is used here, will not change it.
Sorry for confusing you. I tried to modify the comments. Please check it.